src/main/java/duke/duke/Deadline.java

```diff
@@ -0,0 +1,40 @@
+package duke.duke;
```

Comment:
May want to consider moving the different files into relevant folders? E.g. Tasks and subtasks into a "Tasks" folder, then subsequently package them accordingly for better organization!